chore: update opentelemetry java instrumentation documentation (#3747)
SumoLogic/sumologic-documentation: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/java/traceid-spanid-injection-into-logs-configuration.md

    </configuration>
    ```

:::note

> For more details please visit [Logger MDC auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/release/v1.32.x/docs/logger-mdc-instrumentation.md).

Reviewer: ```suggestion
For more details, refer to the [Logger MDC auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/release/v1.32.x/docs/logger-mdc-instrumentation.md).
```
